packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -704,7 +736,10 @@ export class GarbageCollector implements IGarbageCollector {
      * We current write the entire GC state in a single blob. This can be modified later to write multiple
      * blobs. All the blob keys should start with `gcBlobPrefix`.
      */
-    public summarize(): ISummaryTreeWithStats | undefined {
+    public summarize(
+        fullTree: boolean,
+        trackState: boolean,
+    ): ISummaryTreeWithStats | ISummaryTreeHandleWithStats | undefined {
```

Comment:
This should return `ISummarizeResult`. The caller should then add whatever this returns to the summary tree.